docs/b2b-edition/specs/storefront/storefront/company.yaml

```diff
@@ -96,3932 +86,817 @@ paths:
-                  message:
-                    type: string
-                    minLength: 1
-                  data:
-                    type: array
-                    uniqueItems: true
-                    minItems: 1
-                    items:
-                      type: object
-                      properties:
-                        fieldName:
-                          type: string
-                          minLength: 1
-                          description: Field name of one extra field
-                        labelName:
-                          type: string
-                          minLength: 1
-                          description: Label name of one extra field
-                        dataType:
-                          type: string
-                          minLength: 1
-                          description: Type of this field
-                        isRequired:
-                          type: string
-                          minLength: 1
-                          description: If this fields is required
-                      required:
-                        - fieldName
-                        - labelName
-                        - dataType
-                        - isRequired
-                required:
-                  - code
-                  - message
-                  - data
-              examples:
-                example-1:
-                  value:
-                    code: 200
-                    message: SUCCESS
-                    data:
-                      - fieldName: company_tax_exempt_code
-                        labelName: Company Tax Exempt Code
-                        dataType: '0'
-                        isRequired: '0'
-          description: ''
-          headers: {}
-      summary: Get Company Extra Fields
-      tags:
-        - Company
-      operationId: get-companies-extra_fields-storefront
-      description: |-
-        Get company extra fields.
-        Equivalent Storefront GraphQL API Query: `companyExtraFields`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
-    parameters: []
-  '/customers/{userId}/companies':
+                allOf:
+                  - $ref: "#/components/schemas/responseObject"
+                  - type: object
+                    properties:
+                      data:
+                        $ref: "#/components/schemas/extraFields_FULL"
+                      code:
+                        type: number
+                        default: 200
+  /customers/{userId}/companies:
+    parameters:
+      - $ref: "#/components/parameters/userId"
     get:
-      deprecated: false
+      tags:
+        - Companies
+        - Users
+      summary: "Get a Company by User ID"
+      operationId: getcompanybyuserid
       description: |-
-        Get company information by user ID.
+        Returns information for the Company account that corresponds to the entered user ID.
```

Comment:
"entered" seems like odd terminology to use in regard to the URL segment of an API endpoint